roles/linux_ansible_init/files/ansible-init.py

```diff
@@ -103,3 +109,9 @@ def ansible_exec(cmd, *args, **kwargs):
             "127.0.0.1,",
             playbook["name"]
         )
+
+
+SENTINEL = pathlib.Path("/var/lib/ansible-init.done")
+
+SENTINEL.parent.mkdir(mode = o755, parents = True, exist_ok = True)
+SENTINEL.touch(mode=0o644)
```

Comment:
Maybe add a final log statement after to say that `ansible-init` completed successfully